android/app/src/main/java/app/organicmaps/base/BaseMwmFragmentActivity.java

```diff
@@ -41,7 +42,9 @@ public abstract class BaseMwmFragmentActivity extends AppCompatActivity
   protected final void onCreate(@Nullable Bundle savedInstanceState)
   {
     super.onCreate(savedInstanceState);
-    EdgeToEdge.enable(this, SystemBarStyle.dark(Color.TRANSPARENT));
+    int nightMask = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
```

Comment:
1.
https://github.com/organicmaps/organicmaps/blob/d1c932fed48208fd43371b88d6e622d0a05741a9/android/sdk/src/main/java/app/organicmaps/sdk/util/Utils.java#L351-L355

2. Can we set SystemBarStyle in `MwmTheme.MainActivity`?
```xml
Day
<item name="android:statusBarColor">@android:color/transparent</item>
<item name="android:windowLightStatusBar">true</item>
Night
<item name="android:statusBarColor">@android:color/transparent</item>
<item name="android:windowLightStatusBar">false</item>
```